src/ucp/api/ucp.h

```diff
@@ -649,12 +649,12 @@ typedef enum {
  * The enumeration allows specifying which fields in @ref ucp_am_handler_param_t
  * are present. It is used to enable backward compatibility support.
  */
-typedef enum {
-    UCP_AM_HANDLER_ATTR_FIELD_ID       = UCS_BIT(0), /**< id field */
-    UCP_AM_HANDLER_ATTR_FIELD_FLAGS    = UCS_BIT(1), /**< flags field */
-    UCP_AM_HANDLER_ATTR_FIELD_CB       = UCS_BIT(2), /**< cb field */
-    UCP_AM_HANDLER_ATTR_FIELD_ARG      = UCS_BIT(3)  /**< arg field */
-} ucp_am_handler_attr_field;
+enum ucp_am_handler_params_field {
+    UCP_AM_HANDLER_PARAM_FIELD_ID      = UCS_BIT(0), /**< id field */
+    UCP_AM_HANDLER_PARAM_FIELD_FLAGS   = UCS_BIT(1), /**< flags field */
+    UCP_AM_HANDLER_PARAM_FIELD_CB      = UCS_BIT(2), /**< cb field */
+    UCP_AM_HANDLER_PARAM_FIELD_ARG     = UCS_BIT(3)  /**< arg field */
```

Comment:
Since this top level documentation we have to be a bit more verbose.
```
The flag indicates if the active message id is set
The flag indicates if the active message flags is set
The flag indicates if the active message callback function is set
The flag indicates if the active message callback arguments are set
```

Reply:
i'd suggest to take other field mask bits doc was example

Reply:
We have detailed descriptions in the corresponding param structure.
@yosefe, what fields do you mean as an example?

Reply:
ok, i see in most cases field mast bits have less documentation, it's all in the parms struct
maybe `Indicates that @ref ucp_am_handler_params_t.flags field is valid`

Reply:
I like the idea of `@ref`